FNPlugin/Collectors/AtmosphericIntake.cs

```diff
@@ -61,6 +61,8 @@ public override void OnStart(PartModule.StartState state)
             jetTechBonusPercentage = 1 + (jetTechBonus / 10.736f);
         }
 
```

Comment:
Ignore this - it is debug code where I'm trying to debug another issue. AtmosphericIntake seems to be drawing much less than it should. The reported flow in the context menu is much less than it actually receives. Haven't tracked that down yet.